nix_update/update.py

```diff
@@ -174,6 +174,9 @@ def update_version(
         if package.parsed_url.netloc == "github.com":
             _, owner, repo, *_ = package.parsed_url.path.split("/")
             package.diff_url = f"https://github.com/{owner}/{repo}/compare/{package.rev}...{new_version.rev or new_version.number}"
+        elif package.parsed_url.netloc in ["codeberg.org", "gitea.com", "notabug.org"]:
```

Comment:
Maybe we can detect if a remote is a gitea or gitlab automatically but probably something for another PR.

Reply:
would be nice to do this detection only once, but yeah I agree this diserves separate PR